recipes/gcc/gcc-common.inc

```diff
@@ -137,13 +137,15 @@ do_configure_args ?= ""
 addhook set_configure_arch to post_recipe_parse after arch_update
 def set_configure_arch(d):
     args = []
-    for arg_name in ('arch', 'cpu', 'tune'):
+    for arg_name in ('cpu', 'arch', 'tune'):
         if not d.get('ARCH_HAS_WITH_' + arg_name.upper()):
             continue
         arg_value = d.get('TARGET_M' + arg_name.upper())
         if not arg_value:
             continue
         args.append('--with-%s=%s'%(arg_name, arg_value))
+        if "cpu" in arg_name and "arm" in arg_value:
```

Comment:
Could you add a comment on what this line achieves, so that the when the next guy comes around here doesn't end up with to many question marks?